hack/generate-deploy-scripts.sh

```diff
@@ -89,11 +89,12 @@ controller:
     externalTrafficPolicy: Local
 
     annotations:
-      service.beta.kubernetes.io/aws-load-balancer-backend-protocol: http
```

Comment:
is this really not needed anymore?

Reply:
This is an implementation detail for (legacy) in-tree load balancer controller. Defining `aws-load-balancer-backend-protocol: http` for "network load balancer" (NLB) does not make any effect — the controller provisions the listeners of type TCP or SSL.

I couldn't find any documentation that explains this, but I'm referring to this chunk of code https://github.com/kubernetes/legacy-cloud-providers/blob/release-1.22/aws/aws.go#L3941-L3971